win32/src/job_object_helper.c

```diff
@@ -11,7 +11,130 @@
 
 IMPLEMENT_MOCKABLE_FUNCTION(, int, job_object_helper_limit_resources, uint32_t, percent_physical_memory, uint32_t, percent_cpu)
```

Comment:
>int, job_object_helper_limit_resources, uint32_t, percent_physical_memory, uint32_t, percent_cpu [](http://example.com/codeflow?start=30&length=96)

again me, some comments that nobody likes.

This function does 2 things: set memory and CPU.

Imagine a situation where only 1 is in need of adjustment. As it is written now, the user of this module MUST rememeber the OTHER value what it was set to before.

Suggestion is to split in 2 separate APIs.

Reply:
In retrospect, yes, I should have done that. Maybe for the next version.